src/Crypto/pbc-cffi.lisp

```diff
@@ -121,17 +121,37 @@ THE SOFTWARE.
 
 ;; -----------------------------------------------------------------------
 
-(cffi:define-foreign-library libpbc
- (:darwin #.(concatenate 'string 
-		       (namestring (asdf:system-relative-pathname 'emotiq "../var/local/lib"))
-		       "/libLispPBCIntf.dylib"))
- (:linux #.(concatenate 'string 
-		     (namestring (asdf:system-relative-pathname 'emotiq "../var/local/lib"))
-		     "/libLispPBCIntf.so"))
- (t (:default "libLispPBCIntf"))
- )
-
-(cffi:use-foreign-library libpbc)
+;; The Lisp runtime load might help us do this differently, but explicit
+;; initialization is much easier to understand
+
+(defun load-dev-dlls ()
+  "loads the DLLs (.so and .dylib) at runtime, from pre-specified directories"
+  (cffi:define-foreign-library
+   libpbc
+   (:darwin #.(concatenate 
```

Comment:
@guitarvydas Better to use `cl:merge-pathnames` than to convert out to namestring.